app/src/reports/VariableDisparityReport.tsx

```diff
@@ -116,6 +109,7 @@ function VariableDisparityReport(props: {
           </Grid>
           <Grid item xs={props.vertical ? 12 : 6}>
             <DisparityBarChartCard
+              key={variableConfig + "race_and_ethnicity"}
```

Comment:
If you add an object to a string won't this just turn into "[Object object] race_and_ethnicity"? I think you'd want variableConfig.variableId

Reply:
oh wow i'm surprised it was working